src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -510,6 +512,51 @@ g.test('copy_within_same_texture')
     );
   });
 
+// TODO(jiawei.shao@intel.com): test 'stencil-only' is also valid for the texture format 'stencil8'
+g.test('copy_aspects')
+  .params(
+    params()
+      .combine(
+        poptions('formatAndValidAspects', [
+          { format: 'rgba8unorm', validAspects: ['all'] },
+          { format: 'depth24plus-stencil8', validAspects: ['all'] },
+          { format: 'depth32float', validAspects: ['all', 'depth-only'] },
+        ] as const)
+      )
+      .combine(poptions('sourceAspect', ['all', 'depth-only', 'stencil-only'] as const))
+      .combine(poptions('destinationAspect', ['all', 'depth-only', 'stencil-only'] as const))
+  )
+  .fn(async t => {
+    const { formatAndValidAspects, sourceAspect, destinationAspect } = t.params;
+
+    const kTextureSize = { width: 16, height: 8, depth: 1 };
+
+    const srcTexture = t.device.createTexture({
+      size: kTextureSize,
+      format: formatAndValidAspects.format,
+      usage: GPUTextureUsage.COPY_SRC,
+    });
+    const dstTexture = t.device.createTexture({
+      size: kTextureSize,
+      format: formatAndValidAspects.format,
+      usage: GPUTextureUsage.COPY_DST,
+    });
+
+    let isSuccess = false;
+    for (const aspect of formatAndValidAspects.validAspects) {
+      if (sourceAspect === aspect && destinationAspect === aspect) {
```

Comment:
This would compute isSuccess incorrectly e.g. for `format='depth32float';sourceAspect='all';destinationAspect='depth-only'`

Reply:
I am not sure if in this situation we can use either ```all``` or ```depth-only``` on the depth-only formats? In current SPEC it is not very clear about this part.

Reply:
The intent was that it would be valid. The set of aspects in the copy view must be the same, which is true here.

Reply:
Oh. Sorry for my misunderstanding.